Werewolf for Telegram/Werewolf Control/Commands/AdminCommands.cs

```diff
@@ -305,7 +305,7 @@ public static void SetLink(Update update, string[] args)
             }
 
             var link = args[1].Trim();
-            if (!Regex.IsMatch(link, @"^(https?:\/\/)?t(elegram)?\.me\/joinchat\/([a-zA-Z0-9_\-]+)$"))
+            if (!Regex.IsMatch(link, @"^(https?:\/\/)?t(elegram)?\.me\/(\+|joinchat\/)([a-zA-Z0-9_\-]+)$"))
```

Comment:
Looks good now :)
We will merge it when the bot's back running on the server, since it's currently having trouble.